### ISSUE TITLE
Improve visibility of text in shop page

### ISSUE THREAD

**Issue by SaiTejaswaniBikkasani** [Anushkabh/krishiconnect#483]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Hey @Anushkabh 

The text in the "Oil Seeds Fruits" section is currently not visible, likely due to a color contrast issue or font size/color choice. This makes it difficult for users to read and understand the content of this section. We need to update the styling to ensure that the text is easily readable and accessible to all users. This may involve adjusting the font color, background color, or font size to improve readability.

Currently, the page looks like:
![Screenshot 2024-06-03 223013](https://github.com/Anushkabh/krishiconnect/assets/159511364/e7ff6deb-e115-453d-8743-4bb28a1b5ffe)

After making the changes and adding the color:

![Screenshot 2024-06-03 223453](https://github.com/Anushkabh/krishiconnect/assets/159511364/b564f17f-de94-4893-98f5-80b0446e175e)

**Please add the gssoc labels and I would like to work on it under GSSOC'24. Thank you!!**

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue
